include/pf-applications/grain_tracker/distributed_stitching.h

```diff
@@ -466,7 +467,7 @@ namespace GrainTracker
 
           if (!local_particle_max_values.empty())
             particle_max_values[unique_id] =
-              local_particle_max_values[particle_id];
+              local_particle_max_values[local_id];
```

Comment:
This is the bug fix?

Reply:
Yep. I just added a few lines for better readability.